linkml_runtime/utils/schemaview.py

```diff
@@ -141,6 +141,9 @@ def is_absolute_path(path: str) -> bool:
     """
     if path.startswith("/"):
         return True
+    if "://" in path:
+        return True
```

Comment:
cause of the remote import issue!

The import code would benefit from being reworked so that the types are worked out early instead of having several different places where the same or similar checks are made.